src/injector/injector_config.rs

```diff
@@ -82,3 +83,27 @@ pub struct Timespec {
     pub sec: i64,
     pub nsec: i32,
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
+pub enum MistakeType {
+    Zero,
+    Random
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct MistakeConfig {
+    pub class: MistakeType,
```

Comment:
Is `filling` easier to understand?

These configuration will be entered directly by users, so we'd better make it clearer.

Reply:
indeed